scripts/codegen.py

```diff
@@ -98,5 +119,35 @@
         legacy_source,
     )
     legacy_source = re.sub(r"= WebClient\(", "= LegacyWebClient(", legacy_source)
+    legacy_source = re.sub(r"^from slack_sdk.web.chat_stream import ChatStream\n", "", legacy_source, flags=re.MULTILINE)
+    legacy_source = re.sub(r"(?s)def chat_stream.*?(?=def)", "", legacy_source)
```

Comment:
🌲 note: The `LegacyWebClient` won't find this `chat_stream` since responses from the various APIs can't be parsed quick - these are either `async` or `sync`. Instead we might recommend updating to the latest version.

🔗 https://docs.slack.dev/tools/python-slack-sdk/v3-migration